rbac/api/serializers.py

```diff
@@ -56,6 +56,9 @@ def extract_header(request, header):
         Encoded(str): Base64 header
         Decoded(dict): Identity dictionary
     """
+    if header not in request.META:
```

Comment:
unrelated, I just noticed the server blows up with a KeyError 500 if the identity header is not present in the request, so I wanted to catch that